src/backend/functions/change-requests.ts

```diff
@@ -72,7 +73,11 @@ const changeRequestTransformer = (
     ),
     implementedChanges: changeRequest.changes.map((change) => ({
       ...change,
-      wbsNum
+      wbsNum: {
```

Comment:
Since we're making multiple wbsNums here as well, I would copy the `wbsNumOf` function from another file like projects.ts, don't think it can go to utils yet since it has its arg type of WBS_Element which is from prisma, if that were changed it might go in utils.